markdown/inlinepatterns.py

```diff
@@ -207,6 +207,10 @@ def __init__(self, pattern, markdown_instance=None):
         if markdown_instance:
             self.markdown = markdown_instance
 
+    def getExcludes(self):
```

Comment:
Any reason why this is a method? The end user of API (extension developer) is defining a list, not "getting" the list. The fact that the list is being retrieved is not relevant to the user. Rather, the user is interested in defining what to exclude. Why not a property `exclude = [...]`? Does it even need to be specific to an instance of the class?

Reply:
Something like `exclude` is fine (maybe something a bit more unique in case existing extensions happen to use that name for something else).  We can make the default a tuple to be safe so no one will accidentally modify the base list and ripple through all instances that use the base.  That will force people to override the default opposed to appending to it.

To be safe, we can probably just wrap the treeprocessor access with a try/except to be safe as well.